src/guides/v2.3/rest/tutorials/bulk-configurable-product/define-config-product-options.md

```diff
@@ -21,6 +21,8 @@ contributor_link: http://comwrap.com/
 
  The `POST async/bulk/V1/configurable-products/bySku/options` call assigns the specified `attribute_id` to be the configurable attribute.
 
+ Please note that the `attribute_id` is correct, the value can be different on different installations. How to get correct `attribute_id` see [Get the list of attributes defined in an attribute searchCriteria]({{ page.baseurl }}/rest/tutorials/configurable-product/plan-product.html#get-attributes) for more information.
```

Comment:
This is a confusing statement. If it needs to be clear that the 141 value is an example, then use the SAME method as  on the page : https://devdocs.magento.com/guides/v2.4/rest/tutorials/configurable-product/plan-product.html#get-attributes

Suggested change : 

{:.bs-callout-warning}
The attribute ID and value numbers might be different on your installation. Check the values carefully before using them in your calls. To get the correct `attribute_id` see [Get the list of attributes defined in an attribute searchCriteria]({{ page.baseurl }}/rest/tutorials/configurable-product/plan-product.html#get-attributes) for more information.

Reply:
Thank you @BarnyShergold BarnyShergold, I have fixed. Please check.